utilities/logger/src/LoggerInterface.ts

```diff
@@ -0,0 +1,9 @@
+interface LoggerInterface {
```

Comment:
In the rules you were drafting we were saying that interface names should not be used: `ILogger` or `LoggerInterface` etc. let's keep this standard.